tasks/bal_sorswap.ts

```diff
@@ -1,21 +1,25 @@
+/* eslint-disable prefer-const */
 /*
     This code is heavily inspired by: https://github.com/balancer-labs/balancer-sor/blob/master/test/testScripts/swapExample.ts
 */
 
 import type { SwapInfo } from '@balancer-labs/sor';
-import { scale, SOR, SwapTypes } from '@balancer-labs/sor';
+import { bnum, scale, SOR, SwapTypes } from '@balancer-labs/sor';
 import { getBalancerContract } from '@balancer-labs/v2-deployments';
+import type { BigNumberish } from '@ethersproject/bignumber';
+import { Contract } from '@ethersproject/contracts';
 import type { BaseProvider } from '@ethersproject/providers';
+import type { SignerWithAddress } from '@nomiclabs/hardhat-ethers/signers';
 import { BigNumber } from 'bignumber.js';
 import { task } from 'hardhat/config';
 import type { HardhatRuntimeEnvironment } from 'hardhat/types';
 
 enum Networks {
-  MAINNET = 1,
+  mainnet = 1,
```

Comment:
This is the code I borrowed from where MAINNET is capitalized is here: https://github.com/balancer-labs/balancer-sor/blob/master/test/testScripts/swapExample.ts

You might be encountering this problem because you're hitting the 'real' forked environment, instead of the simple mocked up environment we have. It's likely correct we'll need to switch to lowercase. Hopefully will know more later.